grails-app/services/org/pih/warehouse/core/IdentifierService.groovy

```diff
@@ -326,9 +326,9 @@ class IdentifierService {
      *          - empty productIdentifierFormat (will be generated basing on the openboxes.identifier.product.format)
      * */
     String generateProductIdentifier(ProductType productType) {
-        if (productType?.id == ConfigurationHolder.config.openboxes.identifier.defaultProductType.id) {
+        if (productType?.id == Holders.config.openboxes.identifier.defaultProductType.id) {
             // If product type is systems default then generate basing on the generator type
-            def generatorTypeConfig = ConfigurationHolder.config.openboxes.identifier.productCode.generatorType
+            def generatorTypeConfig = Holders.config.openboxes.identifier.productCode.generatorType
```

Comment:
Do we have defaults for these in application.yml? 

We had an issue on the Dominica server a few weeks ago (I think I finally fixed it last week). It feels like we should discuss the expected vs actual behavior for the identifier config as part of a tech huddle.

Reply:
Yeah, it's SEQUENCE by default, needs to be changed according to needs. But I believe now everything works fine in that matter (in grails 1 instances)